Firmware build incompatible with NOP()-defining header (SDCC 3.7.0)

When building with Fedora 29 / sdcc-3.7.0-1.fc29:
```
sdcc -Iinclude -DBUILD_VERSION=`../revision.sh` -DIMME -c cc1111_aes.c
In file included from include/chipcon_usb.h:4,                                                   from include/cc1111.h:48,
                 from include/cc1111_aes.h:7,                                                    from cc1111_aes.c:2:
include/global.h:84:1: warning: "NOP" redefined
In file included from /usr/share/sdcc/include/mcs51/cc1110.h:29,
                 from include/cc1111.h:5,
                 from include/cc1111_aes.h:7,
                 from cc1111_aes.c:2:
/usr/share/sdcc/include/mcs51/compiler.h:88:1: error: this is the location of the previous definition
```

The NOP-macros are, as one might guess, exactly the same.  (pull request soon)

--- a/firmware/include/cc2531.h
+++ b/firmware/include/cc2531.h
@@ -46,7 +46,12 @@ SFRX(USBF5,     0x622A);        // Endpoint 5 FIFO
 // USB activities
 #define USB_ENABLE_PIN              P1_0
 //#define USB_ENABLE_PIN              P1_1
+
+// NOP macro which may be defined elsewhere
+#ifndef NOP
 #define NOP()                       __asm; nop; __endasm;
+#endif
+
 #define USB_DISABLE()               SLEEP &= ~SLEEP_USB_EN;
 #define USB_ENABLE()                SLEEP |= SLEEP_USB_EN;
 #define USB_RESET()                 USB_DISABLE(); NOP(); USB_ENABLE();

--- a/firmware/include/global.h
+++ b/firmware/include/global.h
@@ -81,7 +81,11 @@ extern __xdata u8 ledMode;
 #else
     #define USB_ENABLE_PIN              P1_0
 #endif
+
+// NOP macro which may be defined elsewhere
+#ifndef NOP
 #define NOP()                       __asm; nop; __endasm;
+#endif
 
 // USB data buffer
 #define BUFFER_AVAILABLE		0x00

